src/pages/Wallet/WalletHomePage.tsx

```diff
@@ -145,6 +146,11 @@ const WalletHomePage = () => {
     [client, loadedTxList, setLoadedTxList, setSnackbarMessage, wallet]
   )
 
+  const refreshData = async () => {
```

Comment:
There's no awaited calls in this function, so no need for the `async` keyword.

Reply:
Could be useful to properly await results though, in order to display the refresh animation while loading.

Reply:
Yeah, I need to work on loading animations in general.

Reply:
I will remove the `async` keyword for now, and in a next PR I'll introduce some loading animations, after deciding with you how we want to approach it (loading animation per component that displays data, or a global loader?)